index.html

```diff
@@ -6,7 +6,7 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0">
     <meta name="description" content="Tizen Wearable Web Basic Template" />
 
-    <title>Tizen Wearable Web Basic Application</title>
+    <title>Time to Learn</title>
 
     <link rel="stylesheet" type="text/css" href="css/style.css" />
    <!-- <script data-main="js/build/main-built.js" src="js/lib/require.js"></script> -->

```

Comment:
` <script data-main="js/build/main-built.js" src="js/lib/require.js"></script> `
Should also be here, uncommented, since the version on Github should be deployable